scss/tech.scss

```diff
@@ -0,0 +1,53 @@
+%tech-card-item {
+  color: $mine-shaft;
+  font-family: $font-stack;
+  font-size: $text-medium;
+  position: relative;
+  text-align: center;
+}
+
+.tech-card:before {
+  border: 1px solid $earls-green;
+  border-radius: $tech-card-border-radius;
+  content: ' ';
+  height: 100%;
+  left: -$tech-card-box-overlap;
+  position: absolute;
+  top: -$tech-card-box-overlap;
+  width: 100%;
+  z-index: -1;
+}
+
+.tech-card {
+  align-items: center;
+  background: $earls-green;
+  border-radius: $tech-card-border-radius;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
```

Comment:
this is not suported in most browsers. leave the default one(flex-start) instead. you want the space between elements to be generated by margins and not automatically